packages/manager/apps/web/client/app/domain/domain.service.js

```diff
@@ -1460,6 +1473,19 @@ angular.module('services').service(
       });
     }
 
+    /**
+     * Terminate DNS Anycast
+     * @param {string} serviceName
+     */
+    terminateDnsAnycast(serviceName) {
+      return this.OvhHttp.post(
+        `/domain/zone/${serviceName}/anycast/terminate`,
```

Comment:
why not using ovh api services?

Reply:
Entire domain has apis defined directly. Whole domain has to to be moved to API services. May be we can take this up as refactor work.